lms/envs/aws.py

```diff
@@ -123,6 +123,11 @@
 
 CELERYBEAT_SCHEDULE = {}  # For scheduling tasks, entries can be added to this dict
 
+############################# Persistent Grades ####################################
+
+# Queue to use for updating persistent grades
+RECALCULATE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
@jibsheet there are several other queues specified in this file, do you want me to update those locations as well?